src/ctapipe_io_nectarcam/containers.py

```diff
@@ -10,6 +10,15 @@
     'NectarCAMCameraContainer',
 ]
 
+class NectarCAMDataStreamContainer(Container):
+    """
+    Container for the Fields that are from the DataStream information (new from v6)
+    """
+    telescope_id = Field(-1,"telescope id")
+    sb_id = Field(-1,"No idea what this is")
```

Comment:
ask Dirk?

Reply:
`sb_id` is the scheduling block ID that you get from ACADA.  This should not be stored here in any case, but in your EventSource's `scheduling_blocks` property which should be a `dict[sb_id, ctapipe.containers.SchedulingBlockContainer]`, similar to the `observation_blocks` property for `obs_ids`.   Most of what is in DataStream is not really applicable to a single telescope operating without ACADA, so I don't think you need this container, unless you are using it to temporarily store some information before setting up the EventSource. 

If you don't have an sb_id, you can just set it to some dummy value or to the same value as the obs_id, as we do in the SimTelEventSource currently.

Reply:
Thanks for the clarification on those variables. 
The container class is there to store temporary the information. I decided to go for a container to do as the other class which allow to have it along the data for debugging purpose.